src/rulesets/oas2/index.ts

```diff
@@ -1,5 +1,5 @@
-import { readRulesFromRulesets } from '../reader';
+import { readRuleset } from '../reader';
 
 export { commonOasFunctions as oas2Functions } from '../oas';
```

Comment:
Do we need these files at all? If we keep them do we still need `commonOasFunctions`? I thought this PR would remove them.

Reply:
We can remove them, but if we do so, this will be breaking.